jujugui/static/gui/src/app/components/entity-files/_entity-files.scss

```diff
@@ -0,0 +1,15 @@
+.entity-files {
+    ul {
```

Comment:
I'm not sure I buy that 100%. We have two lists, `entity-files__links` and `entity-files__files`. I could setup a class (separate from their BEM designation) that is shared by both, but that's duplicative of the HTML element they already share… that is, you'd end up with `<ul className="list">`. The benefit (more resilient to markup changes) doesn't seem strong enough to warrant the duplication. Furthermore, the styles specified in these blocks are very specific to the HTML element here - that is, they're overriding default styles. If we changed markup, we likely wouldn't need these styles any more.